connectors/src/connectors/microsoft/lib/graph_api.ts

```diff
@@ -48,10 +48,7 @@ export async function getFilesAndFolders(
     nodeType === "drive"
       ? `${parentResourcePath}/root/children`
       : `${parentResourcePath}/children`;
-  const res = await client
-    .api(endpoint)
-    .select("id,name,createdDateTime,file,folder")
-    .get();
+  const res = await client.api(endpoint).get();
```

Comment:
maybe :
```suggestion
  const res = await getItem(client, endpoint);
```
?

Reply:
In graph_api I would leave it as is
I had added getItem for external use (clean boundary between "MS stuff" and "Dust stuff") but not even sure it's needed